dashboard/app/controllers/blocks_controller.rb

```diff
@@ -0,0 +1,37 @@
+class BlocksController < ApplicationController
```

Comment:
This controller is just providing basic CRUD functionality, so I'd like to stick pretty close to rails defaults/conventions. The only thing I really changed was using the block `name` instead of its `id` for url paths.